### ISSUE TITLE
kwdict operator not implemented

### ISSUE THREAD

**Issue by bnmnetp** [skulpt/skulpt#332]
The following test fragment fails:

```python
coord = {'latitude': '37.24N', 'longitude': '-115.81W'}

def x(latitude=1, longitude=1):
   print latitude, longitude

x()
x(**coord)

UNCAUGHT EXCEPTION: ExternalError: AssertionError: Failure: kwdict not implemented; on line 7
```

**Comment by waywaaard** [comment]
Do you have more information about the proper implementation? In ```Sk.misceval.applyOrSuspend``` the ```kwdict``` is passed and has the values as expected - however no unwrapping is implemented.

- map values from the dict to the arguments (only possible with co_varnames enabled?)
- if dict contains more keys as co_varnames and function does not expect ```**kwargs``` will throw and error
- same but with ```**kwargs``` will move the key/value pairs that are no covarnames to the ```kwargs```

**Comment by bnmnetp** [comment]
I do not have any more information about the proper implementation.  I have not looked into it very much.  However, we can't make it rely on ``co_varnames,`` because things like the string format function do not have a predefined set of keywords.  It might need to look more like ``dict(foo=1, bar=2, ...)``.

**Comment by bnmnetp** [comment]
Or nevermind.  Previous to suspensions dict(**foo) would return an empty dictionary. Now it returns the same error.  Keyword arguments are not supported for dict.

However, I think that there is a link to co_varnames because if the dictionary contains a key that is not a valid keyword argument Python throws an error.

**Comment by meredydd** [comment]
Now fixed in #573.
